android/src/main/java/com/reactnativeteadssdkmodule/RNInReadAdPlacement.kt

```diff
@@ -0,0 +1,198 @@
+package com.reactnativeteadssdkmodule
+
+import android.os.Handler
+import android.os.Looper
+import android.util.Log
+import com.facebook.react.bridge.*
+import com.facebook.react.modules.core.DeviceEventManagerModule
+import tv.teads.sdk.*
+import tv.teads.sdk.renderer.InReadAdView
+import java.util.*
+
+
+class RNInReadAdPlacement(reactContext: ReactApplicationContext) :
+  ReactContextBaseJavaModule(reactContext) {
+
+  override fun getName(): String {
+    return "RNInReadAdPlacement"
+  }
+
+  private fun sendEvent(reactContext: ReactContext, eventName: String, params: WritableMap?) {
```

Comment:
Once you implement tests I suggest you to add in your coverage this listener bridge to check if spreading events from native goes to RN consumer as expected